sphinx4-core/src/main/java/edu/cmu/sphinx/frontend/feature/LiveCMN.java

```diff
@@ -167,8 +169,7 @@ public Data getData() throws DataProcessingException {
             while (initialList.size() < initialCmnWindow) {
                 input = getPredecessor().getData();
                 initialList.add(input);
-                if (input instanceof SpeechEndSignal
```

Comment:
Should not matter will remove diff at that line for you next.

What about the real change made, is that one looking good?  Is there a better way we should consider for addressing those cases?